components/OrchestratorList/index.tsx

```diff
@@ -111,6 +114,13 @@ const OrchestratorList = ({
     () => numeral(Number(principle) || 150).format("0a"),
     [principle]
   );
+  const { data: bondingManagerAddress } = useBondingManagerAddress(); 
+  const { data: treasuryRewardCutRate = BigInt(0.0) } = useContractRead({
```

Comment:
I now set the default value to `BigInt(0.0)`, but this behaviour could be improved.